tasks2.js

```diff
@@ -0,0 +1,184 @@
+   
+   console.log(quoteMotherTeresa);
+    // 3. Check if typeof '10' is exactly equal to 10. If not make it exactly equal.
+    let num = '10'
+
+    console.log(num ===10);
+
+    let numInt = +num
+    console.log(numInt);
+
+// 4. Check if parseFloat('9.8') is equal to 10 if not make it exactly equal with 10.
+    let float = '9.8'
+
+    console.log(float===10);
+
+    let numberInt = parseInt(float)
+
+    console.log(numberInt+1);
+
+// 7. Check if 'on' is found in both python and jargon
+    let python = 'python'
+    let jargon = 'jargon'
+
+    console.log(python.includes('on'));
+    console.log(jargon.includes('on'));
+
+// 8. I hope this course is not full of jargon. Check if jargon is in the sentence.
+    let sentenceTwo = 'I hope this course is not full of jargon'
+
+    console.log(sentenceTwo.includes('jargon'));
+
+// 9. Generate a random number between 0 and 100 inclusively.
+    let randomNumber = Math.random(0-100)
+    let numBtnZeroAndHundred = randomNumber*99
+    console.log(numBtnZeroAndHundred);
+
+// 10. Generate a random number between 50 and 100 inclusively.
+function getRandomNumberFifty() {
+    return Math.floor(Math.random() * 101);
+}
+
+const randomNumber5 = getRandomNumberFifty();
+console.log(randomNumber5);
+
+// 11. Generate a random number between 0 and 255 inclusively.
+    function getRandomNumber() {
+    return Math.floor(Math.random() * 256);
+}
+
+const randomNum = getRandomNumber();
+console.log(randomNum);
+
+
+/* 13. Use console.log() and escape characters to print the following pattern.
+
+1 1 1 1 1
+2 1 2 4 8
+3 1 3 9 27
+4 1 4 16 64
+5 1 5 25 125*/
+
+
+// 14. Use substr to slice out the phrase because because because from the following sentence:'You cannot end a sentence with because because because is a conjunction'
+    console.log(rule.substr(31,24))
+
+/* Exercises: Level 3 */
+//1. 'Love is the best thing in this world. Some found their love and some are still looking for their love.' Count the number of word love in this sentence.
+    const loveTwo = 'Love is the best thing in this world. Some found their love and some are still looking for their love.';
+    const loveCount = (loveTwo.match(/love/gi) || []).length; // Use /gi to match case-insensitively
+    console.log(loveCount); // Output: 3
+
+    
+//2. Use match() to count the number of all because in the following sentence:'You cannot end a sentence with because because because is a conjunction'
+    const sentenceBecause = 'You cannot end a sentence with because because because is a conjunction';
+    const becauseCount = (sentenceBecause.match(/because/gi) || []).length; // Use /gi to match case-insensitively
+    console.log(becauseCount); // Output: 3
+    
+//3. Clean the following text and find the most frequent word (hint, use replace and regular expressions).
```

Comment:
Have you not done this task?

Reply:
that was some challenging for me. I haven't an idea how to solve it